firestore.rules

```diff
@@ -0,0 +1,11 @@
+rules_version = '2';
```

Comment:
One final thing to call out: these are the rules that dictate the conditions for accessing documents in Firebase. This document only dictates what the emulators will do locally, but for now, it's exactly the same thing as on prod.

Here's what these rules mean:
- for all the documents in the "profiles" collection, we only allow reads and writes if the user is authenticated AND the profile's `userUid` matches their user's id
- for all the documents in other collections, we disable read and writes